src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogMessage.cs

```diff
@@ -0,0 +1,58 @@
+using Serilog.Events;
+using Newtonsoft.Json;
+
+namespace Serilog.Sinks.Datadog.Logs
+{
+    /// <summary>
+    /// DatadogMessage sent to logs-backend formatted in JSON.
+    /// </summary>
+    public class DatadogMessage
+    {
+        /// <summary>
+        /// Datadog Integration name.
+        /// </summary>
+        private const string DDSource = "csharp";
+
+        /// <summary>
+        /// Log Event.
+        /// </summary>
+        [JsonProperty("log_event")]
+        public LogEvent Event { get; private set; }
+
+        /// <summary>
+        /// Integration name.
+        /// </summary>
+        [JsonProperty("ddsource")]
+        public string Source { get; private set; }
+
+        /// <summary>
+        /// Service name.
+        /// </summary>
+        [JsonProperty("service")]
+        public string Service { get; private set; }
+
+        /// <summary>
+        /// Custom tags.
+        /// </summary>
+        [JsonProperty("ddtags")]
+        public string Tags { get; private set; }
+
+        private static readonly JsonSerializerSettings settings = new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore };
+
+        public DatadogMessage(LogEvent logEvent, string service, string tags)
+        {
+            Event = logEvent;
+            Source = DDSource;
+            Service = service;
+            Tags = tags;
+        }
+
+        /// <summary>
+        /// Returns the JSON representation of the message.
+        /// </summary>
+        public override string ToString()
```

Comment:
I'm not sure if it makes that much of a difference, but I've never seen the `ToString` function used for serialization. It's usually meant as a short summary of your object that can be printed by the debugger and serialization is done by calling `JsonConvert.SerializeObject` on the object itself, which makes the serialization format explicit

Also having a dependency on `Newtonsoft.Json` in `ToString` could mess with the debugger if there was an issue loading the `Newtonsoft.Json` dependency.